config/application.yml.example

```diff
@@ -58,11 +58,6 @@ development:
   system_satisfaction_survey: false
   editable_statuses: { 14: ['first_draft', 'draft', 'submitted', nil, 'get_a_cost_estimate', 'awaiting_pi_approval'],
                      48: ['draft', 'submitted'] }
```

Comment:
I think we need to leave the navbar links configurable in the yml file. Some institutions may not want a fulfillment link and the SPARCAdmin link will be going away when we go live with our bootstrap branch. It also makes it easy to point these different places for testing.